tests/app/data/test_schema_validation.py

```diff
@@ -148,6 +164,18 @@ def _parse_id_values(self, schema_json, ignored_keys):
                     if isinstance(schema_item, dict):
                         yield from self._parse_id_values(schema_item, ignored_keys)
 
+    def _parse_alias_values(self, schema_json, ignored_keys):
```

Comment:
How would you do this?
The way I just changed it to (_parse_values)... makes the test fail for both alias and id if there is only one duplicate